Mods/intact_mods.txt

```diff
@@ -19,4 +19,10 @@ TG   L or I
 MT   Unlocalized
 MM   -113.08406397853
 CF   C-6 H-11 O-1 N-1
-//
\ No newline at end of file
+//
+ID   Sulfate Adduct
```

Comment:
To be determined by Jacob haha I don't really see a 98.06 peak in unlabeled but if you see it in neucode you might want to add it just to ensure those relations are formed? Unless we don't believe it enough to include that peak in families